CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -1678,10 +1679,22 @@ CFStringRef CFTimeZoneCopyAbbreviation(CFTimeZoneRef tz, CFAbsoluteTime at) {
 }
 
 Boolean CFTimeZoneIsDaylightSavingTime(CFTimeZoneRef tz, CFAbsoluteTime at) {
-    CFIndex idx;
     __CFGenericValidateType(tz, CFTimeZoneGetTypeID());
+#if TARGET_OS_WIN32
+    UErrorCode status = U_ZERO_ERROR;
+    UCalendar *ucal = __CFCalendarCreateUCalendar(NULL, CFSTR("C"), tz);
+    if (ucal == NULL) {
+      return false;
+    }
+    ucal_setMillis(ucal, (at + kCFAbsoluteTimeIntervalSince1970) * 1000.0, &status);
+
+    UBool isDaylightTime = ucal_inDaylightTime(ucal, &status);
+    return isDaylightTime;
```

Comment:
This should be converted from the ICU Boolean to the CoreFoundation Boolean type

Reply:
Not sure `return isDaylightTime ? TRUE : FALSE` is the preferred way. But should be safest though.